src/store/contacts/selector.ts

```diff
@@ -0,0 +1,9 @@
+import { RootState } from '..';
+import { createSelector } from '@reduxjs/toolkit';
+
+export const contacts = (state: RootState) => state.contacts.contacts;
+export const total = (state: RootState) => state.contacts.total;
+export const contactsStore = createSelector([contacts, total], (storeItems, storeTotal) => ({
```

Comment:
Let's call it more clear contacts = {data, total or contacts, total}

Reply:
fix: renamed to data, total